src/components/Task/ExistingTask/TaskCard/index.jsx

```diff
@@ -1,22 +1,31 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
 import ActionButtonGroup from "components/ActionButtonGroup";
 import { formatDate } from "utils/helper";
+import { deleteTask } from "store/actions/todo";
 import styles from "components/Task/ExistingTask/TaskCard/index.module.scss";
 
-function TaskCard({ createdAt, title }) {
+function TaskCard({ taskId, createdAt, title }) {
```

Comment:
component name `TaskCard` should provide the context for prop names. prefixing with `task` is redundant.

```suggestion
function TaskCard({ id, createdAt, title }) {
```